Bottom: aee2dbf960bd44d7cc7cad3b80ad9226e76fe117
Top:    771c126eb52bcdc062cc1d77a4fe074280d26cec
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-28 03:40:53 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 3362da6..6fa8575 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -324,13 +324,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
